cpp/src/IceLocatorDiscovery/PluginI.cpp

```diff
@@ -430,50 +431,33 @@ LocatorI::LocatorI(const string& name,
     {
         _retryDelay = chrono::milliseconds::zero();
     }
-
```

Comment:
Moved the initialization of `_lookups` to  `setLookupReply` called from the `Plugin::initialize` to avoid the optional reply proxies in the vector.